### ISSUE TITLE
testing sigmoid vs soft max in dataset1

### ISSUE THREAD

**Issue by laurentperrinet** [CONECT-INT/2023_CENTURI-SummerSchool#8]
from

> In the multiclass case, the training algorithm uses the one-vs-rest (OvR) scheme if the ‘multi_class’ option is set to ‘ovr’, and uses the cross-entropy loss if the ‘multi_class’ option is set to ‘multinomial’. (Currently the ‘multinomial’ option is supported only by the ‘lbfgs’, ‘sag’, ‘saga’ and ‘newton-cg’ solvers.)

it would be useful to test   ‘ multinomial’ (sigmoid) vs ‘ovr’ (softmax)

**Comment by laurentperrinet** [comment]
checkout https://sebastianraschka.com/blog/2022/losses-learned-part1.html for a nice introduction...
